### PR TITLE
WordPress 5.7 Compatibility

### LINKED ISSUE #5998: WordPress 5.7 Compatibility
## Feature Description

Tracking ticket for WordPress 5.7 compatibility ([planned release in March](https://make.wordpress.org/core/5-7/))

Before release of WordPress 5.7:

* [ ] Bump `Tested up to` in `readme.txt`
* [ ] Run E2E tests against 5.7 RC
* [ ] Update staging environments to use 5.7 RC

After release of WordPress 5.7:

* [ ] Update `php-stubs/wordpress-stubs`
  * [ ] Remove error suppression added in #5997

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #5998

### CODE REVIEW

File: .github/workflows/continuous-integration-e2e.yml [hunk 1]
```diff
@@ -16,13 +16,18 @@ jobs:
...
+    continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
         # TODO: add back Firefox once support is more mature.
         include:
           - browser: 'chrome'
             wp: 'latest'
             coverage: true
+          - browser: 'chrome'
+            wp: '5.7-RC2'
```
Comment (swissspidy) [inline .github/workflows/continuous-integration-e2e.yml]: We're already at RC 3.

Given that 5.7 stable is slated for today, we can probably just wait another day until "latest" becomes 5.7 automatically.

Comment (spacedmonkey) [reply .github/workflows/continuous-integration-e2e.yml]: There wasnt a blog post about that it seems. I have removed.

**Comment by swissspidy** [comment]
If https://github.com/php-stubs/wordpress-stubs/pull/16 gets merged soon we could update that package in this PR as well.

**Review by swissspidy — APPROVED** [review]

**Comment by spacedmonkey** [comment]
This PR is looking good. Let's merge it.

**Comment by swissspidy** [comment]
> This PR is looking good. Let's merge it.

As soon as everything is green

**Pull request merged**